lms/envs/common.py

```diff
@@ -361,7 +361,7 @@
     # There is also an advanced setting in the course module.  The
     # feature flag and the advanced setting must both be true for
     # a course to use saved grades.
-    'ENABLE_SUBSECTION_GRADES_SAVED': False,
+    'ENABLE_SUBSECTION_GRADES_SAVED': True,
```

Comment:
That's a good point - I've been patching it for tests that use this functionality, but it might be better to just have it on globally for testing.